docs/models_list.rst

```diff
@@ -30,13 +30,15 @@ list.actions
 
 - ``label`` is the name of the action.
 - ``action`` is the last portion of a URL, which is used to perform the action.
+- ``useModal`` optionally render the results in a modal view.
```

Comment:
Can we change this to just `modal`, which is more inline with other current DSLs in Linz (such as overview actions and list actions).